src/lib.rs

```diff
@@ -138,11 +139,15 @@ impl PrivateKeyRotationCheckResult {
         }
     }
 
-    //    pub fn group_rotation_needed() -> Option<Vec<GroupId>> {
-    //        unimplemented!()
-    //    }
+    pub fn group_rotation_needed(&self) -> Vec<GroupId> {
```

Comment:
I think the return type here should match `rotations_needed`. That is field is `pub` so that callers can directly match on it if they want to. 

I think returning `Option<Vec1<GroupId>>` here would be good.